Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -60,6 +60,23 @@
     auto* targetNode = nodeForID(nodeID);
     if (auto* node = dynamicDowncast<ScrollingTreeScrollingNode>(targetNode))
         node->handleWheelEventPhase(phase);
+
+    switch (phase) {
+    case PlatformWheelEventPhase::MayBegin:
+    case PlatformWheelEventPhase::Began:
+        scrollingTreeNodeWillStartScroll(nodeID);
```

Comment:
We already hit `ScrollingTree::setUserScrollInProgressForNode()`, and this is adding a parallel mechanism. I think you should trigger the start/stop from `ScrollingTree::setUserScrollInProgressForNode()`.

Reply:
as in call `scrollingTreeNodeWillStartScroll` from `ScrollingTree::setUserScrollInProgressForNode`?